functions.sh

```diff
@@ -181,13 +181,13 @@ eerrorn()
 #
 eerror()
 {
-	if yesno "${EINFO_QUIET}"; then
-		printf " $*\n" >/dev/stderr
+	if yesno "${EERROR_QUIET}"; then
+		return 0
 	else
 		if ! yesno "${RC_ENDCOL}" && [ "${LAST_E_CMD}" = "ebegin" ]; then
-			printf "\n"
+			printf "\n" >&2
```

Comment:
Pretty nit-picky but I wonder if this newline should stay on stdout since its purpose is to complete the message that ebegin wrote. That would adding `&& yesno "${EINFO_QUIET}"` to the condition.